.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
```

Comment:
what is being sourced here?

Reply:
Autogenerated by Husky, it is just sourcing the shell script for commit hooks. The shell script was not committed because of the Husky generated gitignore